autobahn/asyncio/wamp.py

```diff
@@ -172,3 +172,106 @@ def create():
             loop.run_until_complete(protocol._session.leave())
 
         loop.close()
+
+
+# TODO - unify with previous class
+class ApplicationRunnerRawSocket(object):
+    """
```

Comment:
So what would be right solution - extend runner class for broader URI support - tcp:// ...
Or keep it our completely just stick with WS runner?